src/components/Hub/HubImagesList.tsx

```diff
@@ -52,47 +100,65 @@ const HubImagesList = () => {
     getHubImages(getSelectedFilters(filters), searchQuery)
   }
 
+  const handleSortOption = (event: React.ChangeEvent<{ value: string }>) => {
+    setSelectedSortOption(event.target.value)
+  }
+
   return (
     <>
       {isHubImagesLoading ? (
         <SpinningLoader />
       ) : (
-        <Row>
-          <Col md="3">
+        <HubLibraryGrid container>
+          <HubLibraryHeaderGrid item xs={12}>
+            <h3 style={{ margin: 0 }}>
+              Hub / <span style={{ fontWeight: 700 }}>Library</span>
```

Comment:
I think this part is different in the designs. Didn't we decide to go with a tabbed interface?